docs/recipes/next/integrate.md

```diff
@@ -0,0 +1,93 @@
+---
+id: nextjs
+title: Integrate next js with effector
+---
+
+To do this, we will use the native [fork](https://effector.dev/docs/api/effector/fork) method.
+In the __app.tsx_ file, we create our own [Scope](https://effector.dev/docs/api/effector/scope) and wrap the application in a provider from _effector-react/scope_
```

Comment:
```suggestion
To do this, we will use the native [fork](../../../api/effector/fork.md) method.
In the __app.tsx_ file, we create our own [Scope](../../../api/effector/Scope.md) and wrap the application in a provider from _effector-react/scope_
```

It is better to link for internal files, due to build system can validate the links